motor-control/firmware/motor_encoder_hardware.h

```diff
@@ -0,0 +1,19 @@
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+#include <stdbool.h>
```

Comment:
I can't see a use of `bool` type. Are you sure this include is needed?

Reply:
you're right, I removed it. I don't need it. Thanks!